test/compiler/inline.jl

```diff
@@ -1301,7 +1301,8 @@ let src = code_typed1(Tuple{Any}) do x
             DoAllocNoEscapeSparam(x)
         end
     end
-    @test count(isnew, src.code) == 0
+    # FIXME
+    @test_broken count(isnew, src.code) == 0 && count(iscall(DoAllocNoEscapeSparam), src.code) == 0
```

Comment:
This is not because of this PR, but I found that this test case has been broken.
On master:
```julia
julia> src = code_typed1(Tuple{Any}) do x
           for i = 1:1000
               DoAllocNoEscapeSparam(x)
           end
       end
CodeInfo(
1 ─       goto #7 if not true
2 ┄ %2  = φ (#1 => 1, #6 => %9)::Int64
│         Main.DoAllocNoEscapeSparam(x)::Any
│   %4  = (%2 === 1000)::Bool
└──       goto #4 if not %4
3 ─       goto #5
4 ─ %7  = Base.add_int(%2, 1)::Int64
└──       goto #5
5 ┄ %9  = φ (#4 => %7)::Int64
│   %10 = φ (#3 => true, #4 => false)::Bool
│   %11 = Base.not_int(%10)::Bool
└──       goto #7 if not %11
6 ─       goto #2
7 ┄       return nothing
)
```

Reply:
Maybe because of #45062? /cc @ianatol

Reply:
Sure thing, I will take a look today